src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -64,16 +68,11 @@ private void OnParseCommand(string value)
         }
 
         /// <summary>
-        /// Converts an hexadecimal value to an UTF-8 string
+        /// Hex String to String
+        /// input:  48656c6c6f20576f726c6421
+        /// output: Hello World!
         /// </summary>
-        /// <param name="hexString">
-        /// Hexadecimal value to be converted
-        /// </param>
-        /// <returns>
-        /// Returns null when is not possible to parse the hexadecimal value to a UTF-8
-        /// string or when the converted string is not printable; otherwise, returns
-        /// the string represented by the hexadecimal value
-        /// </returns>
+        [ParseFunction("Hex String to String")]
         private string? HexToString(string hexString)
         {
             try
```

Comment:
This can be replaced with `Convert.ToHexString`

Reply:
Already removed `HexToString` `StringToHex`  `NumberToHexToString` `HexToNumber` `Base64ToHexString` `HexScriptsToOpCode`  method